src/controllers/PenaltyDetailsController.ts

```diff
@@ -72,7 +68,10 @@ export class PenaltyDetailsController extends BaseAsyncHttpController {
         const appealObject = extraData
             .chainNullable(data => data.appeals as Appeal)
             .mapOrDefault(changePenaltyIdentifier, Maybe.of({
-                penaltyIdentifier: body
+                penaltyIdentifier: {
```

Comment:
the company number was not being sanitised. I realised we were sanitising the company no on the appeal data in the session rather than the request body